python/practices/practice-3/src/path1/word_e_checker.py

```diff
@@ -0,0 +1,35 @@
+"""
+Count and print percentage word in word list not contains the letter 'e' or 'E'
+"""
+
+
+import os
+
+
+def count_words():
+    """
+    Count and calculate the percentage of words that do not contain 'E' or 'e'
+    in a list of words.
+    """
+
+    words = input("Enter the list of words: ").split()
+    count_word = len(words)
```

Comment:
Update the variable name, such as `total_number_of_words`

Reply:
Done